lib/mollie/payment.rb

```diff
@@ -23,6 +23,7 @@ class Payment < Base
                   :expires_at,
                   :failed_at,
                   :amount,
+                  :amount_captured,
                   :amount_refunded,
```

Comment:
Can you also add the `authorizedAt ` parameter as well?

Reply:
👍